src/Pim/Bundle/ReferenceDataBundle/Doctrine/ORM/Repository/ReferenceDataRepository.php

```diff
@@ -90,6 +90,19 @@ public function findOneByIdentifier($identifier)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function findCodesByIdentifiers(array $referenceDataCodes)
+    {
+        return $this->createQueryBuilder($this->getAlias())
```

Comment:
Yes, but it's used everywhere in the class, I would be consistent with the rest of the class